src/app/pages/service/photos/photos.service.ts

```diff
@@ -0,0 +1,45 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http'
+import { environment } from '../../../environments/environment'
+
+@Injectable({
+    providedIn: 'root'
+})
+
+export class PhotosService {
+
+    constructor(private http: HttpClient) { }
+
+    getPhotos(): Promise<Photo[]> {
+        return this.http.get(`https://${environment.facebookphotosUrl}`, {
+        params: { token: environment.facebookphotosToken }
+    }).toPromise()
+    .then((response: any) =>
+    //is this the correct way to make an api call?
+        FB.api(
+        "/{photo-id}",
+        function (response) {
```

Comment:
Get used to arrow functions will help a lot.
In this case it will be 
```javascript
res => {
  if(res && !res.error) { }
}
```
or 
```javascript
(err, res) => {
  if ( res && !err ) { }
}
```